### PR TITLE
Feature/164077034/get single art

### PR DESCRIPTION
**Pull request by daniellamarr**
#### What does this PR do?
- This PR gets creates a single art page
#### Description of Task to be completed?
- Create a page that displays data of a single art
- Implement logic to get single art details
#### How should this be manually tested?
- Pull this branch of this repository
- Run `npm install`
- Run `npm run start:dev`
- Navigate to `/single-art`
#### Any background context you want to provide?
- N/A
#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/164077034
#### Screenshots (if appropriate)
screen shot 2019-02-25 at 7 18 22 am
screen shot 2019-02-25 at 7 18 25 am

#### Questions:
- N/A

### CODE REVIEW

File: webpack/webpack.dev.js [hunk 1]
```diff
@@ -2,6 +2,11 @@ const merge = require('webpack-merge');
 const common = require('./webpack.common.js');
 
 module.exports = merge(common, {
+  output: {
+    path: `${__dirname}/dist`,
+    publicPath: '/',
```
Comment (julietezekwe) [inline webpack/webpack.dev.js:6]: What is the essence of the other things you added? I think just adding `publicPath` to output should solve this problem. Kindly explain if you have a reason for the filename

File: src/assets/scss/single-art.scss [hunk 2]
```diff
@@ -0,0 +1,243 @@
...
+  width: 30%;
+  float: right;
+}
+
+.art-reactions ul li {
+  display: inline;
+  margin-right: 40px;
+}
+
+.art-reactions ul li span {
```
Comment (wombolo) [inline src/assets/scss/single-art.scss:68]: Kindly take advantage of SASS style rules nesting.
This block can go on line 65 above, without `.art-reactions ul li`

**Review by wombolo — CHANGES_REQUESTED** [review]
I left some feedback

File: webpack/webpack.dev.js (hunk 1 above)
Comment (daniellamarr) [reply webpack/webpack.dev.js:6]: Yeah, my bad.
I would have taken that off, it was just an oversight.
I would change it now.

File: src/components/SingleArt.jsx [hunk 3]
```diff
@@ -0,0 +1,225 @@
...
+    this.props.getSingleArt(slug);
+  }
+
+  /**
+   * @param {function} render
+   *  @returns {JSX} jsx
+   */
+  render() {
+    const { isLoading, details } = this.props.art;
+    return (
```
Comment (andela-dbamidele) [inline src/components/SingleArt.jsx:25]: Please go ahead and break this down into multiple smaller components. This components is too long and hard to follow.

File: src/components/SingleArt.jsx [hunk 4]
```diff
@@ -0,0 +1,225 @@
...
+      </div>
+    );
+  }
+}
+
+SingleArt.propTypes = {
+  getSingleArt: PropTypes.func.isRequired,
+  art: PropTypes.shape({
+    isLoading: PropTypes.bool.isRequired,
+    details: PropTypes.object,
```
Comment (andela-dbamidele) [inline src/components/SingleArt.jsx:212]: **DON'T USE** `PropTypes.object`!

**Review by andela-dbamidele — CHANGES_REQUESTED** [review]
